### ISSUE TITLE
Questions about the results of libSVM

### ISSUE THREAD

**Issue by lzm42** [cjlin1/libsvm#209]
LibSVM version: C++OneClassSVM
Specify parameters: nu and gamma
Call method: Use C++to pass output, C # to call
Dataset invocation method: Train unlabeled datasets and predict anomalies for that dataset
Now I have encountered the following problem:
1. Although nu and gamma have been specified, the results may change during repeated training. It was observed that this is due to changes in the number of iterations, but no parameters that can control the iterations were found
2. When nu=0.05 and gamma=0.15, not only does the number of iterations change, but in most cases, the output of the iter is 0, indicating that there is no training at all
3. At nu=0.05 and gamma=0.15, the number of iterations is 0 and 88, where all results are -1 when item=0 and all results are 1 when item=88
4. Let nu=0.2, gamma=0.15, and the number of iterations be 91, but the results are all -1. It is worth noting that at this point, the auc metric of OCSVM in sklearn has exceeded that version

**Comment by cjlin1** [comment]
Can you show us your data so we can see why results change in repeated 
training?

**Comment by lzm42** [comment]
from sklearn.datasets import laod_wine,load_breast_cancer. I deleted the target columns of two datasets and standardized them

**Comment by cjlin1** [comment]
Could you show us your code?

**Comment by cjlin1** [comment]
Can you show us some exp log. Indeed because we don't have your whole 
code/environment to reproduce what you got, it is difficult to see what 
is going on

**Comment by lzm42** [comment]
We only need data and parameters to reproduce it. We did not make any changes to the code, it was just a simple call, so there are no relevant logs
